src/panel/menu/AppMenu.jsx

```diff
@@ -0,0 +1,62 @@
+/* globals _ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import MenuItem from './MenuItem';
+import Telemetry from 'src/libs/telemetry';
+import { Divider } from 'src/components/ui';
+import { Heart, IconLightbulb, IconEdit, IconApps } from 'src/components/ui/icons';
+import { PINK_DARK, ACTION_BLUE_BASE } from 'src/libs/colors';
+
+const AppMenu = ({ close, openProducts }) => {
+  const navTo = (url, options) => {
+    close();
+    window.app.navigateTo(url, options);
+  };
+
+  return <div className="menu-items">
+    <MenuItem
+      onClick={e => {
```

Comment:
I think we should add an href to `/favs/` here :)

Reply:
I agree this would be a better markup than a link to "#", even if the navigation is done in JS. I'll do it.

Reply:
It's done, thanks!